app/components/views/PrivacyPage/Privacy/PrivacyContent.jsx

```diff
@@ -55,6 +56,14 @@ const PrivacyContent = ({
     {accountMixerError && (
       <div className={style.error}>{accountMixerError}</div>
     )}
+    { accountMixerRunning && (
+      <div className={style.logs}>
+        <div className={style.logsTitle}>
+          <T id="privacy.mixer.logs" m="Logs:" />
```

Comment:
Maybe use a standard `<Subtitle>` for this?

Reply:
changed